src/game_events/manager.cpp

```diff
@@ -178,7 +178,7 @@ void manager::write_events(config& cfg, bool include_nonserializable) const
 		// have been flagged as disabled by this point (such events are disabled before their
 		// actions are run). If a disabled event is encountered outside an event context,
 		// however, assert. That means something went wrong with event list cleanup.
```

Comment:
This comment needs an extra sentence explaining the `include_nonserializable`.

The code change itself seems good giving the timing (I approve of getting it into tonight's build), but IIUC it's a hack of "we know that only the inspector uses this, and it makes sense to allow `[inspect]` at odd times, so avoid the assert".